### DISCUSSION TITLE
Want to add user in AWS Cognito pool

### DISCUSSION THREAD

**Discussion by ashwani-cm** [Budibase/budibase#7747] [Help]
Hello, please help me with how to create a user in AWS cognito pool by using Budibase

Thanks, will be obliged for your help

**Comment by melohagan** [comment]
Hey @ashwani-cm 

I may not understand the question, but having a quick look at Amazon Cognito user pools (https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html) it appears you can use a variety of SSO options.

The Budibase docs for these can be found here: https://docs.budibase.com/docs/authentication-and-sso

If you are looking to use the Cognito API, then you can use the Budibase REST connector: https://docs.budibase.com/docs/rest
You may need to use the OAuthToken to authenticate your requests:
![image](https://user-images.githubusercontent.com/101575380/190096777-6bf9fdce-7f85-4d5e-b33d-f8e8a4b0aa5d.png)
